bezier-rs/docs/interactive-docs/src/App.vue

```diff
@@ -55,6 +62,32 @@ export default defineComponent({
 					// eslint-disable-next-line
 					callback: (): void => {},
 				},
+				{
+					name: "Bezier from points",
+					// eslint-disable-next-line
+					callback: (): void => {},
+					createThroughPoints: true,
+					template: markRaw(SliderExample),
+					templateOptions: { sliders: [{ ...tSliderOptions }] },
+					cubicOptions: {
+						sliders: [
+							{
+								min: 0.01,
+								max: 0.99,
+								step: 0.01,
+								default: 0.5,
+								variable: "t",
+							},
+							{
+								min: 0,
+								max: 100,
+								step: 5,
+								default: 10,
+								variable: "strut",
```

Comment:
What does "strut" mean? The only definition I know for that word is "a rod or bar forming part of a framework and designed to resist compression" which presumably isn't the case here?

Reply:
I named this variable here to be consistent with the function parameter in the Rust code. The variable in the rust function is meant to represent the distance between `e1` and `e2` in [this section of the primer](https://pomax.github.io/bezierinfo/#abc) and on a high level, it can sort of be understood as controlling how wide the resulting cubic curve will be around point `t`. I don't really know what's the best name for this parameter - if you have a better suggestion, please let me know!

Reply:
Update: I've changed this variable name to `midpoint_separation` (suggested by Keavon)